text/0006-shipyard-next-generation.md

```diff
@@ -0,0 +1,315 @@
+# The next generation of Shipyard
+
+The next generation of Shipyard to allow custom delivery/remediation processes.
+
+## Motivation
+
+*There's always room for smart enhancements* - the Shipyard specification is no exception.
+
+The current specification of Keptn's [Shipyard](https://github.com/keptn/spec/blob/0.1.7/shipyard.md) has several limitations and hides details from the end-users. This KEP provides an enhancement of the *Shipyard* format such that it allows end-users to specify *WHAT* should happen in the course of an artifact delivery or remediation process. While it brings flexibility into the definition of the *WHAT*, it is still an opinionated approach with smart defaults.
+
+In addition, please consider [KEP 07: Propose the concept of a Uniform](https://github.com/keptn/enhancement-proposals/pull/7) as it specifies which components responds to certain events/tasks defined within the Shipyard.
+
+## Explanation
+
+:man:/:blonde_woman: For instance, one target persona of this KEP could be an SRE (Site Reliability Engineer), who is responsible for processes and workflows to automate application delivery or remediation operations. 
+
+Before getting started, a brief recap of the important terms and definitions used in this KEP:
+
+- **Project stage:** A project stage (or just stage) defines a logical space in an environment, which has a dedicated purpose for an application in a continuous delivery process. At least one stage must be defined (e.g., for a single stage project), but multiple stages are allowed (e.g., for implementing a staging process using a development, hardening, and production stage). If multiple stages are defined, by default they are ordered according to the shipyard file. However, it is possible to change the default order by defining triggers mentioned below.
```

Comment:
last part of this paragraph: add **as**
*by defining triggers **as** mentioned below*